### PR TITLE
Fix P2P unit tests

### PR DESCRIPTION
**Pull request by fassadlr**

### CODE REVIEW

File: src/Stratis.Bitcoin.Tests/P2P/PeerAddressManagerTests.cs [hunk 1]
```diff
@@ -168,7 +168,7 @@ public void PeerAddressManager_AttemptThresholdTimeNotReached_DoNotReset()
             Assert.Null(savedPeer.LastConnectionSuccess);
             Assert.Null(savedPeer.LastConnectionHandshake);
             Assert.Null(savedPeer.LastSeen);
-            Assert.Equal("127.0.0.1", savedPeer.Loopback.ToString());
+            Assert.Equal("::ffff:127.0.0.1", savedPeer.Loopback.ToString());
```
Comment (dangershony) [inline src/Stratis.Bitcoin.Tests/P2P/PeerAddressManagerTests.cs]: How did this ever pass?

File: src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs [hunk 2]
```diff
@@ -947,8 +943,8 @@ public async void AcceptToMemoryPool_TxMissingInputs_ReturnsFalseAsync()
             // CheckMempoolCoinView !context.View.HaveCoins(txin.PrevOut.Hash)
             bool isSuccess = await validator.AcceptToMemoryPool(state, tx);
             Assert.False(isSuccess, "Transaction with invalid input should not have been accepted.");
-            Assert.Null(state.Error.Code); // No specific error set for this case, only the MissingInputs flag.
-            Assert.Equal(0, state.Error.RejectCode);
+            Assert.Equal("bad-txns-inputs-missing", state.Error.Code);
```
[the remark this answers is not in the document]
Comment (fassadlr) [reply src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs]: Yeah this was your change :) What do you mean test jar?

File: src/Stratis.Bitcoin.Tests/P2P/PeerAddressManagerTests.cs (hunk 1 above)
Comment (fassadlr) [reply src/Stratis.Bitcoin.Tests/P2P/PeerAddressManagerTests.cs]: I dont think it did!

Comment (fassadlr) [reply src/Stratis.Bitcoin.Tests/P2P/PeerAddressManagerTests.cs]: @noescape00 made changes here so loop back is now mapped to IPv6

File: src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (dangershony) [reply src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs]: nothing never mind 👀

**Pull request merged**